README.md

```diff
@@ -8,7 +8,23 @@ Download the system for free from [here](https://github.com/svof/svof/releases)
 See [documentation](https://svof.github.io/svof/) on how to install, use, and take advantage of the powerful system in your scripts.
 
 # Contributing
-See the [developer readme](Developer readme.md) for useful information on the code, acquaint yourself with the [code style](https://github.com/svof/svof/wiki/Code-style), and read the [design principles](https://github.com/svof/svof/wiki/Svof-design-principles).
+
+1. Make the changes in mudlet.
+2. Click Save Profile.
+3. Go to the [Svof Github](https://github.com/svof/svof).
+4. Switch to the in-client-svof branch.
+5. Select Upload files it on the right next to the [green Clone or Download button](https://github.com/svof/svof/upload/in-client-svof). 
+6. Drag the changed .xmls onto the box, or choose them from the file explorer.
+7. Add a title/description.
+8. Check off "Create a new branch for this commit and start a pull request".
+9. Name your branch.
+10. Click propose changes.
```

Comment:
I think here is the step to check for that the changes look legit - only what they changed, and not everything.